applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>Small point, but I think the footnote citations should come before the period and the colon in this cell.</p><p> </p><p>EDIT: actually, I think this point goes for all similar footnotes in this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>